hydroserving/helpers/package.py

```diff
@@ -7,27 +7,55 @@
 from hydroserving.helpers.contract import read_contract_cwd
 
 
-def pack_path(entry):
-    model_dirs = os.path.dirname(entry)
+def get_subfiles(path):
```

Comment:
It can be replaced with oneliner or 
```python
def get_subfiles(path):
    return [os.path.join(dir_name, file) for dir_name, _, files in os.walk(path) for file in files]
def get_subfiles2(path):
    all_files = []
    for dir_name, _, files in os.walk(path):
        all_files.extend(map(lambda f: os.path.join(dir_name, f), files))
    return all_files
```